autopush/router/apnsrouter.py

```diff
@@ -64,6 +64,7 @@ def __init__(self, conf, router_conf, metrics, load_connections=True):
         self.metrics = metrics
         self._base_tags = ["platform:apns"]
         self.apns = dict()
+        self._max_retry = router_conf.get("max_retry", 2)
```

Comment:
This could use an example in the conf file -- which leads me to notice this key sort of clashes with apns' router_conf format (a dict of rel_channel/cert_infos). Probably should be loaded from a separate config knob?

Reply:
Well, this should come out of the router_config dict (`apns_creds`), so there should't be any conflicts. (basically, it's already in a different config knob set). 

Will add to sample config.

Reply:
and, yes, `apns_certs` has been horribly overloaded to include "platform" specific configuration options like this. Legacy, is a thing.

Reply:
max_retry's still assumed to be top level here (alongside the rel_channel names) but it's per rel_channel in the conf. e.g. it would be `router_conf[<some_rel_channel>].get('max_retry')`